### PR TITLE
Fix bug with sac

### PR DESCRIPTION
**Pull request by witwolf**

### CODE REVIEW

File: alf/algorithms/sac_algorithm.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def actor_loss_fn(dqda, action):
             if self._dqda_clipping:
                 dqda = tf.clip_by_value(dqda, -self._dqda_clipping,
                                         self._dqda_clipping)
-            loss = losses.element_wise_squared_loss(
+            loss = 0.5 * losses.element_wise_squared_loss(
```
Comment (emailweixu) [inline alf/algorithms/sac_algorithm.py]: To  be consistent, we should also have 0.5 for ddpg_algorithm

File: alf/examples/sac_pendulum.gin [hunk 2]
```diff
@@ -0,0 +1,44 @@
+
+import alf.algorithms.sac_algorithm
+import alf.utils.common
+
+# environment config
+NUM_PARALLEL_ENVIRONMENTS=1
```
Comment (emailweixu) [inline alf/examples/sac_pendulum.gin]: why only use 1 here?

File: alf/algorithms/sac_algorithm.py (hunk 1 above)
Comment (witwolf) [reply alf/algorithms/sac_algorithm.py]: updated

File: alf/examples/sac_pendulum.gin (hunk 2 above)
Comment (witwolf) [reply alf/examples/sac_pendulum.gin]: Its an appropriate value here .  process communication  would be heavy load for this simple gym env when `ParallelPyEnvironment` is used ,it actually cost less training steps when parallel environments are used but it slows down the training it terms of total time

**Pull request merged**